### PR TITLE
Russian version set up

### PR DESCRIPTION
**Pull request by michaelrambeau**
A PR to check the set up for the Russian version, prepared by @lex111

https://github.com/bestofjs/javascript-risingstars/pull/63

### CODE REVIEW

File: data/allLanguages.json [hunk 1]
```diff
@@ -22,5 +22,9 @@
   {
     "code": "id",
     "text": "Indonesia"
+  },
+  {
+    "code": "ru",
+    "text": "русский"
```
Comment (lex111) [inline data/allLanguages.json]:
```suggestion
    "text": "Русский"
```

**Review by lex111 — COMMENTED** [review]
Looks good! Just fix one small thing, and it's done.

File: data/allLanguages.json (hunk 1 above)
Comment (michaelrambeau) [reply data/allLanguages.json]: Thank you @lex111 done!

Just to be sure, is the bio at the end OK?

image

Comment (lex111) [reply data/allLanguages.json]: Yep, let's merge this.

**Pull request merged**